packages/node_modules/@ciscospark/internal-plugin-board/src/realtime.js

```diff
@@ -148,21 +166,31 @@ const RealtimeService = Mercury.extend({
    */
   connectToSharedMercury(channel) {
     return this.spark.internal.board.registerToShareMercury(channel)
-      .then((res) => {
-        this.boardBindings = [res.binding];
-        this.boardWebSocketUrl = res.webSocketUrl;
-
-        if (!res.sharedWebSocket) {
-          return this.connect()
-            .then(() => {
-              this.isSharingMercury = false;
-              return res;
-            });
+      .then(tap(({binding, webSocketUrl}) => {
```

Comment:
Given that tap will suppress errors, let's try to get in the habit of commenting why we're suppressing errors.